src/main/webapp/js/datahandlers/scholarshipdatahandler.js

```diff
@@ -16,6 +16,7 @@
 
 goog.module('datahandlers.scholarshipdatahandler');
 const {Map: SoyMap} = goog.require('soy.map');
+const {SinglePageDataHandler} = goog.require('datahandlers.singlepagedatahandler');
 const {addSpaceToCamelCase} = goog.require('datahandlers.utils');
 
 const NA = 'N/A';
```

Comment:
Make a map of fields to display names. This way you don't need to do any string manipulations, especially if you're already hardcoding this.

```
const FIELDS = [
  { fieldName:  'academicRequirements', displayName: 'Academic Requirements' },
];
```

Reply:
I added a map